README.md

```diff
@@ -539,6 +540,17 @@ This repository contains a curated list of awesome open source libraries that wi
 * [yellowbrick](https://github.com/DistrictDataLabs/yellowbrick) ![](https://img.shields.io/github/stars/DistrictDataLabs/yellowbrick.svg?style=social) - yellowbrick is a matplotlib-based model evaluation plots for scikit-learn and other machine learning libraries.
 
 
+## Industry Strength RecSys
+* [EasyRec](https://github.com/alibaba/EasyRec) ![](https://img.shields.io/github/stars/alibaba/EasyRec.svg?style=social) - EasyRec is a framework for large scale recommendation algorithms.
+* [Gorse](https://github.com/gorse-io/gorse) ![](https://img.shields.io/github/stars/gorse-io/gorse.svg?style=social) - Gorse aims to be a universal open-source recommender system that can be quickly introduced into a wide variety of online services.
+* [Implicit](https://github.com/benfred/implicit) ![](https://img.shields.io/github/stars/benfred/implicit.svg?style=social) - Implicit provides fast Python implementations of several different popular recommendation algorithms for implicit feedback datasets
+* [LightFM](https://github.com/lyst/lightfm) ![](https://img.shields.io/github/stars/lyst/lightfm.svg?style=social) - LightFM is a Python implementation of a number of popular recommendation algorithms for both implicit and explicit feedback
+* [NVTabular](https://github.com/NVIDIA-Merlin/NVTabular) ![](https://img.shields.io/github/stars/NVIDIA-Merlin/NVTabular.svg?style=social) -  NVTabular is a feature engineering and preprocessing library for tabular data that is designed to easily manipulate terabyte scale datasets and train deep learning (DL) based recommender systems.
+* [Merlin](https://github.com/NVIDIA-Merlin/Merlin) ![](https://img.shields.io/github/stars/NVIDIA-Merlin/Merlin.svg?style=social) - NVIDIA Merlin is an open source library providing end-to-end GPU-accelerated recommender systems, from feature engineering and preprocessing to training deep learning models and running inference in production.
+* [Recommenders](https://github.com/Microsoft/Recommenders) ![](https://img.shields.io/github/stars/Microsoft/Recommenders.svg?style=social) - Recommenders contains examples and best practices for building recommendation systems, provided as Jupyter notebooks. 
```

Comment:
I'm not sur eabou tthis one, it seems its more examples than a framework

Reply:
> I'm not sure about this one, it seems its more examples than a framework

Good question. That remember me https://github.com/EthicalML/awesome-production-machine-learning/pull/294. Do we include production-level benchmark like And benchmarks like [SafePO-Baselines](https://github.com/PKU-MARL/Safe-Policy-Optimization), [RLeXplore](https://github.com/yuanmingqi/rl-exploration-baselines), [D4RL](https://github.com/Farama-Foundation/D4RL), [Metaworld](https://github.com/Farama-Foundation/Metaworld), [Overcooked-AI](https://github.com/HumanCompatibleAI/overcooked_ai) and [OmniSafe](https://github.com/PKU-MARL/omnisafe)? in our list? @axsaucedo

Reply:
If we further check our list, we would find we already include the production-level benchmarks like fairness:
![image](https://user-images.githubusercontent.com/8592144/221969565-dbd84a9b-760e-40fe-8c68-db46cc252de1.png) @axsaucedo

Reply:
Main part of Recommenders is essentially a Python package with SOTA models for RecSys. Here is the [PyPi documentation](https://pypi.org/project/recommenders/), and [here](https://github.com/microsoft/recommenders/blob/main/examples/00_quick_start/dkn_MIND.ipynb) is an example of how it can be used. So it is like scikit-learn for RecSys.

Reply:
This list is for frameworks and not for benchmarks, so if there aer any ones spotted let's remove - this does seem more of a package so I would say it could be kept. If we spot items in the list that are just benchmarks I'd be keen to remove

Reply:
> 

Hi, @axsaucedo I recently think seriously about our list and propose a feasible plan to benchmarks, take a look at https://github.com/EthicalML/awesome-production-machine-learning/pull/370. This seems to solve the current dilemma.

Reply:
As discussed in #370 can be discussed in separate issue, but for now let's remove and explore if readding in the separate issue

Reply:
I have opened a new issue here: https://github.com/EthicalML/awesome-production-machine-learning/issues/386 @axsaucedo

Reply:
Benchmarking is only a part of Recommenders. Other parts (imho more valuable) are APIs for ReqSys models (aka ReqSys scikit-learn), data loads, auto hyperparameters tuning. 
![image](https://user-images.githubusercontent.com/12928778/223242847-7fe805e9-1e5c-4fc9-8119-73353844a229.png)

Reply:
> 

@axsaucedo Maybe we can still use Recommender? @axsaucedo

Reply:
Keen to merge this PR as there's been quite a lot of discussion, let's keep as is + change to "recommender systems" and then explore separate as I'm sure there will be more tools to add in the medium term